inception/embedding_service.py

```diff
@@ -192,6 +194,9 @@ async def generate_text_embeddings(
                 )
             all_embeddings.append(text_embeddings_list)
 
+        end_time = time.time()
+        logger.info(f"Wrap-up took {end_time - embed_time:.2f} seconds")
```

Comment:
These chained timers sort of feel like pointers, but I'll allow it. This isn't a pattern I've seen before.

I can't decide if it's clever (what I first thought), or a mistake because it makes adding more timers difficult (maybe?). I think I'm leaning toward clever. Let's do it.